docs/tutorials/sample-based-quantum-diagonalization.ipynb

```diff
@@ -123,11 +124,11 @@
     "3. For each subset of bitstrings, project the Hamiltonian into the subspace spanned by the corresponding basis vectors (see the [previous section](#quantum-chemistry) for a description of these basis vectors), and compute a ground state estimate of the projected Hamiltonian on a classical computer.\n",
     "4. Update the estimate of the average orbital occupancies with the ground state estimate with the lowest energy.\n",
     "\n",
-    "The SQD workflow is depicted in the following diagram:\n",
+    "### SQD workflow diagram\n",
     "\n",
-    "![sqd_diagram.png](/docs/images/tutorials/improving-energy-estimation-of-a-fermionic-hamiltonian-with-sqd/fd7e816f-4e2e-4dd7-a7da-f71afb9ca68d.avif)\n",
+    "The SQD workflow is depicted in the following diagram:\n",
     "\n",
-    "SQD is known to work well when the target eigenstate is sparse: the wave function is supported in a set of basis states $\\mathcal{S} = \\{|x\\rangle \\}$ whose size does not increase exponentially with the size of the problem."
+    "![sqd_diagram.png](/docs/images/tutorials/improving-energy-estimation-of-a-fermionic-hamiltonian-with-sqd/fd7e816f-4e2e-4dd7-a7da-f71afb9ca68d.avif)"
```

Comment:
The `sqd_diagram.png` in brackets needs to be replaced with alt-text about the diagram, for accessibility - a brief description would be fine

Reply:
Done. Also added alt text to the other pictures.